src/core/int128.d

```diff
@@ -0,0 +1,695 @@
+/* 128 bit integer arithmetic.
+ *
+ * Not optimized for speed.
+ *
+ * Copyright: Copyright D Language Foundation 2022.
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   Walter Bright
+ * Source:    $(DRUNTIMESRC core/_int128.d)
+ */
+
+module core.int128;
+
+nothrow:
+@safe:
+@nogc:
+
+alias I = long;
+alias U = ulong;
+enum Ubits = U.sizeof * 8;
+
+struct Cent
+{
+    U lsl;	// low 64 bits
+    U msl;      // high 64 bits
+}
+
+enum One = Cent(1);
+enum Zero = Cent();
+
+/*****************************
+ * Test against 0
+ * Params:
+ *	c = Cent to test
+ * Returns:
+ *	true if != 0
+ */
+pure
+bool tst(Cent c)
+{
+    return c.msl || c.lsl;
+}
+
+
+/*****************************
+ * Complement
+ * Params:
+ *	c = Cent to complement
+ * Returns:
+ *	complemented value
+ */
+pure
+Cent com(Cent c)
```

Comment:
all these free functions should be operator overloads

Reply:
I decided against that because:
1. better to pass arguments by value rather than by ref
2. easier to interface with C, will probably make them extern (C) functions

Reply:
They don't need to be extern(C), especially with three letter names.